apps/laboratory/tests/shared/utils/project.ts

```diff
@@ -55,7 +56,8 @@ const ETHERS5_DISABLED_TESTS = [
   'smart-account.spec.ts',
   'wallet-features.spec.ts',
   'siwe-email.spec.ts',
-  'siwe-sa.spec.ts'
+  'siwe-sa.spec.ts',
+  'no-socials.spec.ts'
```

Comment:
why disable it?

Reply:
+ ethers5 should have the feature parity with wagmi/ethers

Reply:
Yeah added test for solana as well 🙏